spec/lib/govuk_configuration_spec.rb

```diff
@@ -51,6 +51,28 @@
     end
   end
 
+  describe '#clamscan_path' do
+    context 'when environment includes an ASSET_MANAGER_CLAMSCAN_PATH value' do
+      let(:env) {
+        {
+          'ASSET_MANAGER_CLAMSCAN_PATH' => 'md5',
```

Comment:
Minor: The choice of 'md5' seems odd. I think it might be clearer to use something like 'alternative-path'.